apps/api/src/app/workflows-v2/usecases/post-process-workflow-update/post-process-workflow-update.usecase.ts

```diff
@@ -94,7 +92,10 @@ export class PostProcessWorkflowUpdate {
     return issue.body && Object.keys(issue.body).length > 0;
   }
 
-  private validateSteps(steps: NotificationStepEntity[], _workflowId: string): Record<string, StepIssuesDto> {
+  private validateSteps(
+    steps: NotificationStepEntity[],
+    _workflowId: string
+  ): Record<string, StepIssuesDto> | undefined {
```

Comment:
Nit-picky one:

```suggestion
  ): Record<string, StepIssuesDto> | null {
```

In TS null feels more explicit than undefined as undefined.

Reply:
this gets persisted, I prefer to use undefined here as it goes into mongo, since it's a nit-pick I choose to not take the proposal here